pages/docs/data-ops.mdx

```diff
@@ -23,7 +23,7 @@ Hashboard allows you to represent your resources as configuration files, which c
     href="/docs/data-ops/quickstart"
   >
     Use the same tools that you use to develop your backend pipeline to manage
-    your BI tool
+    your BI tool.
```

Comment:
Our style guide is to not have end stops for titles / bullets etc on the marketing page.  So don't think we need them here since this is a label / title that you would click on

Reply:
Cool, was just going for consistency - I'll remove them here and from the rest then!